agenta-backend/agenta_backend/routers/container_router.py

```diff
@@ -18,12 +17,17 @@
 else:
     from agenta_backend.services.selectors import get_user_and_org_id
 
-if os.environ["FEATURE_FLAG"] in ["cloud", "ee"]:
+if os.environ["FEATURE_FLAG"] in ["cloud"]:
+    from agenta_backend.ee.services import container_manager
+if os.environ["FEATURE_FLAG"] in ["ee"]:
     from agenta_backend.ee.services import container_manager
 else:
     from agenta_backend.services import container_manager
 
 
+if os.environ["FEATURE_FLAG"] in ["oss", "cloud"]:
+    from agenta_backend.services.docker_utils import restart_container
```

Comment:
I would actually create a function restart_container insider the container_manager and call that instead. I don't think the router should import a low-level lib such as docker_utuils directly